index.html

```diff
@@ -0,0 +1,119 @@
+              <p>First Time full course in Arabic language </p>
+        </div>
+        </div>
+      </section>
+   </div>
+  <section id="About" class="main-topic">
+        <h3>Main Topic</h3>
+        <hr>
+        <div class="container item-wraper">
+        <div class="main-topic-item">
+        <img src="assets/img/htmlSide.png" alt="htmlSide">
+            <p class="title">HTML</p>
+            <p class="text">HTML is the standard markup language for Web pages. With HTML you can create your own Website.</p>
+        </div>
+        <div class="main-topic-item">
+            <img src="assets/img/CSS3Side.png" alt="CSS3Side">
+                <p class="title">CSS</p>
+                <p class="text">CSS is the language we use to style an HTML document.CSS describes how HTML elements should be displayed.</p>
+            </div>
+            <div class="main-topic-item">
+                <img src="assets/img/javascript.png" alt="javascript">
+                    <p class="title">JavaScript</p>
+                    <p class="text">JavaScript is the world's most popular programming language. JavaScript is the programming language of the Web.</p>
+                </div>
+                <div class="main-topic-item">
+                    <img src="assets/img/PHPSide.png" alt="PHPSide">
+                        <p class="title">PHP</p>
+                        <p class="text">PHP is a server scripting language, and a powerful tool for making dynamic and interactive Web pages.</p>
+                    </div>
+                    <div class="main-topic-item">
+                        <img src="assets/img/pythonSide.jpeg" alt="pythonSide">
+                            <p class="title">python</p>
+                            <p class="text"> Python is a programming language. Python can be used on a server to create web applications</p>
+                        </div>
+                          </div>
+        <a class="join-button" href="#">Join course</a>
+    </section>
+  <section class="container courses">
+        <h3>Courses</h3>
+        <hr>
+        <div id="courses" class="row  mt-5"></div>
+          <button class="more-btn btn btn-outline-dark" type="submit">more</button>
+  </section>
+  <section>
+    <div class="footer-black">
+      <h3>Partner</h3>
+      <hr>
+      <ul>
+        <li>google</li>
+        <li>facebook</li>
+        <li>twitter</li>
+        <li>microverse</li>
+      </ul>
+    </div>
+    <div class="footer-light">
+             <h3>
+                 MUSTAFA ABBAS
+             </h3>
+             <p >
+              All rights reserved to Mustafa Abbas 
+             </p>
+    </div>
+  </section>
+  <script src='assets/script.js'></script>
+<!-- JavaScript Bundle with Popper -->
+<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.0/dist/js/bootstrap.bundle.min.js" integrity="sha384-U1DAWAznBHeqEIlVSCgzq+c9gqGAJn5c/t99JyeKa9xxaYpSvHU5awsuZVVFIhvj" crossorigin="anonymous"></script>
+</body>
```

Comment:
- [ ] The _partner_ and _footer_ sections are missing in your design in your mobile version. Kindly ensure to improve upon these aspects by adding the _partner_ and _footer_ sections to your mobile version of your design.

**Your mobile bottom part:**
![image](https://user-images.githubusercontent.com/12745474/132610147-40756ca1-fd42-4aa0-a646-316682ba80ac.png)

**The original bottom part:**
![image](https://user-images.githubusercontent.com/12745474/132610415-16677bde-c76a-43a9-a103-d2fc9dbe06f8.png)